pkg/versionstream/resolver.go

```diff
@@ -3,6 +3,8 @@ package versionstream
 import (
 	"fmt"
 
+	"github.com/jenkins-x/jx/v2/pkg/util/errors"
```

Comment:
it would be nice to keep this one consistent with the imports in the other files, pending the outcome of the previous comment :)